submodules/vmbox/vmbox.js

```diff
@@ -338,12 +366,19 @@ define(function(require) {
 					$this.addClass('disabled');
 
 					if (monster.ui.valid(vmboxForm)) {
-						var form_data = monster.ui.getFormData('vmbox-form');
+						var form_data = monster.ui.getFormData('vmbox-form'),
+							$skipInstructionsInput = vmbox_html.find('#skip_instructions_input').val();
 
 						form_data.notify_email_addresses = getRecipients();
 
-						/* self.clean_form_data(form_data); */
+						if (form_data.announcement_only) {
+							form_data.skip_instructions = $skipInstructionsInput === 'true' ? true : false;
+						} else {
+							delete form_data.announcement_only;
+							delete data.data.announcement_only;
```

Comment:
If you check the saving method, you'll notice form data and document data gets merged and goes through a `vmboxNormalizeData` method, which is where we manipulate it to fit API requirements.

This check should be moved there.